src/js/downpour.js

```diff
@@ -1,13 +1,13 @@
-'use strict';
-
-const path = require('path');
-const inform = require('./lib/inform');
-const drizzle = require('drizzle-builder');
-const helpers = require('@cloudfour/hbs-helpers');
-const assembleHelpers = require('handlebars-helpers');
-const ifProd = require('@sparkbox/if-prod');
-const shelljs = require('shelljs');
-const _ = require('lodash');
+import '../css/drizzle.scss';
```

Comment:
I feel this magic line needs a comment. Since there are no tests asserting the correct output of the CSS file, it would be too easy for someone to delete this weird line. That hapless dev might not notice until downstream npm users complained at errors.